hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md

```diff
@@ -123,33 +129,77 @@ Configuration
+  In this example the memory is set in percentage mode, the cpu is set in weight mode and the gpu is set in absolute unit.
+  It is also possible to use to old capacity format, e.g.: `50.0` for percentage, `[memory=1024,vcores=1]` for absolute and `1w` for weight mode.
+  Different modes can be mixed freely in the queue hierarchy.
+
+  The effective min and max resources are calculated based on the queues' capacity configuration and the available cluster resources.
+  Calculating the effective resources is done in the following way, for each defined resource type:
+  1. the configured absolute capacities are allocated to the queues from the available cluster resources
+  2. the remaining cluster resources are allocated for the queues with percentage resource configurations
+  3. the rest of the resources are allocated for the queues with weighted resource configurations
+
+  Example for mixed queue resource allocation using the Universal Capacity Vector format:
+```
+    # Configuration
+    yarn.scheduler.capacity.legacy-queue-mode.enabled = false
+    yarn.scheduler.capacity.root.queues = default, test_1, test_2
+    yarn.scheduler.capacity.root.test_1.queues = test_1_1, test_1_2, test_1_3
+    yarn.scheduler.capacity.root.default.capacity =         [memory=1w, vcores=1w]
+    yarn.scheduler.capacity.root.test_1.capacity =          [memory=16384, vcores=16]
+    yarn.scheduler.capacity.root.test_1.test_1_1.capacity = [memory=50%, vcores=50%]
+    yarn.scheduler.capacity.root.test_1.test_1_2.capacity = [memory=1w, vcores=1w]
+    yarn.scheduler.capacity.root.test_1.test_1_3.capacity = [memory=12288, vcores=12]
+    yarn.scheduler.capacity.root.test_2.capacity =          [memory=75%, vcores=75%]
+
+    # ClusterResources=[32GB 32VCores]  EffectiveMin                     AbsoluteCapacity
+    root.default              4/32      [memory=4096,    vcores=4]       12.5%
+    root.test_1              16/32      [memory=16384,   vcores=16]
+    root.test_1.test_1_1        2/16      [memory=2048,  vcores=2]       6.25%
+    root.test_1.test_1_2        2/16      [memory=2048,  vcores=2]       6.25%
+    root.test_1.test_1_3       12/16      [memory=12288, vcores=12]      37.5%
+    root.test_2              12/32      [memory=12288,   vcores=12]      37.5%
+```
+  Further examples can be found at `TestRMWebServicesCapacitySchedulerMixedMode.java`.
+
+  The capacity for the root queue cannot be configured, it is fixed to 100% (percentage mode).
+
+  The effective capacity, absoluteCapacity and derived properties like the maximumApplications are calculated from the effective resources.
+  If there is no cluster resource, the maximumApplications fallbacks to the configured values.
+
+  Flexible AQC does not support the Universal Capacity Vector format yet: [YARN-11520](https://issues.apache.org/jira/browse/YARN-11520)
```

Comment:
YARN-11520 is under review, so we could wait for it with this update.

Reply:
Removed and updated the flexible aqc part.